corehq/apps/accounting/models.py

```diff
@@ -1026,7 +1031,9 @@ class Subscription(models.Model):
     is_hidden_to_ops = models.BooleanField(default=False)
     skip_auto_downgrade = models.BooleanField(default=False)
 
-    objects = SubscriptionManager()
+    visible_objects = VisibleSubscriptionManager()
+    visible_and_suppressed_objects = models.Manager()
```

Comment:
This basically renames `objects` to `visible_and_suppressed_objects`, right?

Reply:
yeah